tests/python/pytorch/sparse/test_sparse_matrix.py

```diff
@@ -596,14 +596,30 @@ def test_print():
     col = torch.tensor([2, 1, 3]).to(ctx)
     val = torch.tensor([1.0, 1.0, 2.0]).to(ctx)
     A = from_coo(row, col, val)
-    print(A)
+    expect_res = str(
+        """SparseMatrix(indices=tensor([[1, 1, 3],
+                             [2, 1, 3]]),
+             values=tensor([1., 1., 2.]),
+             shape=(4, 4), nnz=3)"""
```

Comment:
```suggestion
        """
SparseMatrix(indices=tensor([[1, 1, 3],
                             [2, 1, 3]]),
             values=tensor([1., 1., 2.]),
             shape=(4, 4), nnz=3)
        """
```

Reply:
Not sure whether this passes the test or not, but looks better in this way.

Reply:
> Not sure whether this passes the test or not, but looks better in this way.

I tried it locally. It does not pass the test, for there will be another '\n' before the expected result, which causes assertion error.

Reply:
I see, then ignore this.